vaxrank/core_logic.py

```diff
@@ -29,6 +29,11 @@
 logger = logging.getLogger(__name__)
 
 
+# Lifted from https://www.python.org/dev/peps/pep-0485/#proposed-implementation
+def _isclose(a, b, rel_tol=1e-09, abs_tol=0.0):
```

Comment:
Probably better to reuse https://docs.scipy.org/doc/numpy/reference/generated/numpy.isclose.html rather than implement this via copy/paste.